app/frontend/src/utils/util.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
This file should be utils.js (plural)
Unneeded whitespace line here.

Reply:
Might just remove this file. See comment below (if so will leave until next PR)

Reply:
This would be one of those fantastic cases where we can use a "functional SFC" and pass in via prop. I think that may make more sense - if nothing else that should be something we can investigate. Here's a pretty good article on it and it has an example SFC that this could match with:
https://dev.to/fdietz/vue-js-functional-components-2495